docs/CONTRIBUTING.md

```diff
@@ -2,9 +2,13 @@
 
 **This document is under review** 
```

Comment:
Do we need to keep this in? I would like to see that the document just reflects the current state.

Reply:
Agreed, if it is better to remove that, then I'll update the PR.

Anyone else have thoughts on the "under review" sentence?

Reply:
Agreed, can be removed.

Reply:
done & pushed